omero/sysadmins/server-upgrade.txt

```diff
@@ -26,10 +26,11 @@ Check prerequisities
 Before starting the upgrade, please ensure that you have obtained all
 the prerequisites for installation, documented for Unix and Windows. In
```

Comment:
Since you've started editting this section, could you link to the relevant prerequisites pages?